examples/resources/astro_api_token/resource.tf

```diff
@@ -0,0 +1,81 @@
+resource "astro_api_token" "example" {
+  name        = "api token"
+  description = "api token description"
+  type        = "ORGANIZATION"
+  roles = [{
+    "role" : "ORGANIZATION_OWNER",
+    "entity_id" : "clx42kkcm01fo01o06agtmshg",
+    "entity_type" : "ORGANIZATION"
+  }]
+  expiry_period_in_days = 30
+}
+
+resource "astro_api_token" "example" {
```

Comment:
lets have these two have different terraform variable names instead of both being `"example"`